mono/mini/aot-compiler.c

```diff
@@ -7861,6 +7870,10 @@ mono_aot_parse_options (const char *aot_options, MonoAotOptions *opts)
 			opts->mode = MONO_AOT_MODE_FULL;
 			opts->llvm = TRUE;
 			opts->llvm_only = TRUE;
+		} else if (str_begins_with (arg, "llvm-plt")) {
+			opts->mode = MONO_AOT_MODE_FULL;
```

Comment:
I don't think we should have this as an option, maybe as a MONO_DEBUG option, since we don't want to support both configurations.